OMNA/presence/presence-service-description-registry.html

```diff
@@ -193,7 +193,6 @@
            </tr>
            <tr>
               <td class="TableP1big" style="height: 20px">CPM Standalone Capabilities</td>
-               <td class="TableP1big">CPM Standalone Capabilities</td>
```

Comment:
Please can you make sure the indentation layout is correct?

Use ctrl-K then ctrl-D (or Edit Menu -> Format Document)

Thanks